pkg/thumbnail/imgsource/webdav.go

```diff
@@ -27,12 +28,17 @@ type WebDav struct {
 func (s WebDav) Get(ctx context.Context, file string) (image.Image, error) {
 	u, _ := url.Parse(s.baseURL)
 	u.Path = path.Join(u.Path, file)
+	fmt.Printf("url: %s", u.String())
 	req, err := http.NewRequest(http.MethodGet, u.String(), nil)
 	if err != nil {
 		return nil, fmt.Errorf("could not get the image \"%s\" error: %s", file, err.Error())
 	}
 
+	// FIXME: make this configurable!!
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
you could add your certificate to your the list of trusted certs

Reply:
Ideally we should address this before merging

Reply:
do we have this pattern somewhere, like config switches ? or rely on running debug mode vs non-debug ?

Reply:
found it, ocis-proxy has some config pattern. I'll use the same.